src/_layouts/layout-side-nav.html

```diff
@@ -12,27 +12,27 @@
 
 {# Set default value for nav_items array. #}
 {% set nav_items = [(
-    "/docs/sheer-layouts/layout-side-nav/",
-    "",
-    "Please review the instructions in _layouts/layout-side-nav.html for setting up the side nav."
+    '/docs/sheer-layouts/layout-side-nav/',
```

Comment:
This is actually a dead link till we get the docs site setup. Perhaps needs a `TODO` added.